### PR TITLE
update to 0.79.3

### PR DESCRIPTION
**Pull request by Furtif**

### CODE REVIEW

**Comment by Gegy** [comment]
Can you test this with updated UK25, version and hashing endpoint?
From HatchingEgg on Discord:
```
- VersionInt is: 7701
- UK25 is: 0xA50D4ECF47B25C0D
- Endpoint is: 0.77.1/1.47.1 (api/v147_1/hash)
```

**Comment by Furtif** [comment]
is same for 79.3 Endpoint is: 0.77.1/1.47.1 (api/v147_1/hash) version int 7903

File: library/build.gradle [hunk 1]
```diff
@@ -3,8 +3,8 @@ buildscript {
         jcenter()
     }
     dependencies {
-        classpath 'com.google.protobuf:protobuf-gradle-plugin:0.7.7'
-        classpath "com.jfrog.bintray.gradle:gradle-bintray-plugin:1.7.1"
+        classpath 'com.google.protobuf:protobuf-gradle-plugin:0.8.2'
```
Comment (Gegy) [inline library/build.gradle:6]: Is updating protobuf in the scope of this PR?

Comment (Furtif) [reply library/build.gradle:6]: i have updated only if not ok make you values

**Comment by Gegy** [comment]
Complete update handled in #938

**Pull request closed without merging**